docs/data/charts/styling/ColorTemplate.js

```diff
@@ -5,22 +5,45 @@ import Stack from '@mui/material/Stack';
 import TextField from '@mui/material/TextField';
 import MenuItem from '@mui/material/MenuItem';
 import Typography from '@mui/material/Typography';
+import globalChance from 'chance';
+
+const chance = globalChance();
```

Comment:
This is flaky in Argos CI, e.g. https://app.argos-ci.com/mui/mui-x/builds/11605/49763612. I think that we could solve this like in the data grid:

https://github.com/mui/mui-x/blob/38733c626ea74819a6b0d90b7a2baa7ea8a024f1/packages/grid/x-data-grid-generator/src/services/random-generator.ts#L20-L22